BoincRpc/RpcClient.cs

```diff
@@ -80,6 +80,12 @@ public async Task<bool> AuthorizeAsync(string password)
             }
         }
 
+        /// <summary>
+        /// Exchange version info with the client.
+        /// This request not requiring authentication.
```

Comment:
Could you fix the grammar in all comments where this sentence appears? It should be "This request does not require authentication." or "This request requires to be authenticated.".

Reply:
Fixed.

Reply:
Done.